src/scripts.js

```diff
@@ -236,24 +176,35 @@ const makeBtnsClickable = () => {
       showRecipeInfo(event)
     })
   })
-  // Add event listeners to favorite icons
-
-  // favoriteBtns.forEach(btn => {
-  //   btn.addEventListener('click', () => {
-  //     addToUserFaves(event)
-  //   })
-  // })
+  favoriteBtns.forEach(btn => {
+    btn.addEventListener('click', () => {
+      addToUserFaves(event)
+    })
+  })
 }
 
 // To be utilized for userFavorites, and load random user
 
-// const addToUserFaves = (event) => {
-//
-// }
-//
-// const getRandomIndex = (array) => {
-//   return Math.floor(Math.random() * array.length);
-// }
+const addToUserFaves = (event) => {
+    console.log(event.target.id);
```

Comment:
I'm thinking we can implement something similar for deleting from user favorites